src/test/java/unit/GcloudBuildsSubmit.java

```diff
@@ -0,0 +1,84 @@
+package unit;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import bio.terra.cli.businessobject.Context;
+import bio.terra.cli.businessobject.Server;
+import harness.TestCommand;
+import harness.TestUser;
+import harness.baseclasses.SingleWorkspaceUnit;
+import java.io.File;
+import java.io.IOException;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+/** Tests for the `terra gcloud builds submit` commands. */
+@Tag("unit")
+public class GcloudBuildsSubmit extends SingleWorkspaceUnit {
+  private TestUser shareUser;
+
+  @BeforeAll
+  @Override
+  protected void setupOnce() throws Exception {
+    super.setupOnce();
+
+    TestCommand.runCommandExpectSuccess("workspace", "set", "--id=" + getUserFacingId());
```

Comment:
28-32 aren't needed, right?

Reply:
Hi Melissa. The workspace set is required because the gcs bucket specify need the workspace information

Reply:
Discussed offline. Test is setting workspace on line 57. That is what most CLI tests do, they set workspace in beginning.

Lines 28-32 don't look like they're needed. They set up `shareUser` but `shareUser` isn't used by test. Please try deleting.